Directory.Build.targets

```diff
@@ -211,9 +211,13 @@ Expected to exist:
       Command="curl -L https://github.com/getsentry/sentry-cocoa/releases/download/$(CocoaVersion)/Sentry-Dynamic.xcframework.zip -o $(SentryCocoaCache)Sentry-Dynamic-$(CocoaVersion).xcframework.zip" />
 
     <Exec
-      Condition="!Exists('$(SentryCocoaCache)Sentry-Dynamic.xcframework')"
+      Condition="!Exists('$(SentryCocoaCache)Sentry-Dynamic.xcframework') AND !$([MSBuild]::IsOSPlatform('Windows'))"
       Command="unzip -o $(SentryCocoaCache)Sentry-Dynamic-$(CocoaVersion).xcframework.zip -d $(SentryCocoaCache)" />
 
+    <Exec
+      Condition="!Exists('$(SentryCocoaCache)Sentry-Dynamic.xcframework') AND $([MSBuild]::IsOSPlatform('Windows'))"
+      Command="powershell -Command &quot;Expand-Archive -Path '$(SentryCocoaCache)Sentry-Dynamic-$(CocoaVersion).xcframework.zip' -DestinationPath '$(SentryCocoaCache)' -Force&quot;" />
+
```

Comment:
there're a lot of powershell scripts in the repo anyway - maybe worth using only that on all platforms?

Reply:
Well.. actually, fair enough!